src/RequestGroupMan.h

```diff
@@ -195,6 +198,18 @@ class RequestGroupMan {
 
   bool removeReservedGroup(a2_gid_t gid);
 
+  bool getOptimizeSimultaneousDownloads() const
+  {
+    return optimizeSimultaneousDownloads_;
+  }
+
+  void setOptimizeSimultaneousDownloads(bool flag)
+  {
+    optimizeSimultaneousDownloads_=flag;
+  }
+
+  bool setOptimizeSimultaneousDownloads();
```

Comment:
Usually, we use setSomething to set parameter to private field.  Since this function has no parameters, perhaps, something like setupOptimizeSimultaneousDownloads would sound more like it.